slack-api-client/src/main/java/com/slack/api/SlackConfig.java

```diff
@@ -88,13 +88,48 @@ public void setMethodsConfig(MethodsConfig methodsConfig) {
         public void setProxyUrl(String proxyUrl) {
             throwException();
         }
+
+        @Override
+        public void setHttpClientCallTimeoutMillis(Integer httpClientCallTimeoutMillis) {
+            throwException();
+        }
+
+        @Override
+        public void setHttpClientWriteTimeoutMillis(Integer httpClientWriteTimeoutMillis) {
+            throwException();
+        }
+
+        @Override
+        public void setHttpClientReadTimeoutMillis(Integer httpClientReadTimeoutMillis) {
+            throwException();
+        }
```

Comment:
👍🏻 It's really nice that you expose all 3 (read/write/call) to allow developers to fine-tune the settings for their situation.